assignment.js

```diff
@@ -113,6 +136,62 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+//Creating necessary variables
+let playerAChoice = 'Rock'
+let playerBChoice = 'Paper'
+
+//Function for generating numbers 1-3
+function randomNumberGenerator() {
+  return Math.floor(Math.random() * 3) + 1
+}
+
+//Function for assigning a number to Rock, Paper or Scissors
+function assignChoice() {
+  //Assigns the randomNumberGenerator function to randomNumber variable
+  let randomNumber = randomNumberGenerator()
+  //If statement to assign number 1-3 to a game choice
+  if(randomNumber === 1) {
+    return 'Rock'
+  } else if (randomNumber === 2) {
+    return 'Paper'
+  } else {
+    return 'Scissors'
+  }
+}
+
+//Function for the game
+function theGame() {
+  //Assigns the random choice to each player
+  let playerAChoice = assignChoice()
+  let playerBChoice = assignChoice()
+
+  console.log('Players get ready! Make your choices.')
+
+  //Prints out the results to verify winner
+  console.log(`Player A chooses: ${playerAChoice}`)
+  console.log(`Player B chooses: ${playerBChoice}`)
+
+  //If statement to determine all choice options and results
+  if (playerAChoice === playerBChoice) {
+    console.log('This is a tie game.')
+  } else if (playerAChoice === 'Rock' && playerBChoice === 'Paper') {
+    console.log('Player B Wins!')
+  } else if (playerAChoice === 'Rock' && playerBChoice === 'Scissors') {
+    console.log('Player A Wins!')
+  } else if (playerAChoice === 'Paper' && playerBChoice === 'Rock') {
+    console.log('Player A Wins!')
+  } else if (playerAChoice === 'Paper' && playerBChoice === 'Scissors') {
+    console.log('Player B Wins!')
+  } else if (playerAChoice === 'Scissors' && playerBChoice === 'Rock') {
+    console.log('Player B Wins!')
+  } else if (playerAChoice === 'Scissors' && playerBChoice === 'Paper') {
+    console.log('Player A Wins!')
+  }
+}
+
+//Starts the game
+theGame()
+
```

Comment:
@cswebdesign Great Job here with your solution for the game. You broke down the game and separated out parts of your logic into reusable functions. Good Job using the `assignChoice` function to pick a random choice for each player and use conditionals to determine the winner of the game. I would recommend adding one more conditional to return an error if the player's choice is incorrect.
Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L129)